docs/editor/artificial-intelligence.md

```diff
@@ -90,6 +90,14 @@ Copilot will provide a suggestion like the following:
 
 Use `kbstyle(Tab)` to accept the suggestion.
 
+### Alternative and partial solutions
+
+For any given input, GitHub Copilot may offer multiple suggestions. When Copilot offers a suggestion, you can hover over the suggestion to see the GitHub Copilot command palette for choosing suggestions:
+
+![JavaScript ghost text suggestion](images/artificial-intelligence/copilot-hover-highlight.png)
+
+In the image above, Copilot presents 3 suggestions. You can accept the entire suggestion with `kbstyle(Tab)`, or only part of the suggestion with `Ctrl + ->`.
+
```

Comment:
Let's also mention that you can switch between suggestions, or you can use the keybinding `alt/option + ]` without the hover. FYI command name is `editor.action.inlineSuggest.showNext`

Reply:
Makes sense, added info on this! Let me know what you think. And thanks for the command name :)